test/simulation.cpp

```diff
@@ -0,0 +1,43 @@
+#include <random>
```

Comment:
What exactly is this simulation? A stress test of the std priority queue?

Reply:
oops, an unnecessary